### ISSUE TITLE
Improve accessibility experience

### ISSUE THREAD

**Issue by pyscripter** [pyscripter/SynEdit#53]
Currently screen readers just read the whole content of the Synedit.

Instead, as with other editors, one would like to, for example, have the selected text or the current line read, when the selection or the current line changes.

For that, the IAccessible interface needs to be implemented.

**Comment by pyscripter** [comment]
Implemented using the newer [Microsoft UI Automation](https://learn.microsoft.com/en-us/windows/win32/winauto/entry-uiauto-win32).

Full support for Screen readers such as Narrator and NVDA.

**Comment by JaFi-cz** [comment]
It looks like unit SynAccessibility is missing in github

**Comment by pyscripter** [comment]
Oops!  It has now been added. Thanks
